### PR TITLE
Create new file 0000-01-02-AmkaE.md

### PR DESCRIPTION
**Pull request by itzAmka**
Created a new file called 0000-01-02-AmkaE.md in _post folder.

### CODE REVIEW

File: _posts/0000-01-02-AmkaE.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```
Comment (AmkaE) [inline _posts/0000-01-02-AmkaE.md:5]: "Your text"

**Review by itzAmka — COMMENTED** [review]
reviewing changes

**Review by itzAmka — COMMENTED** [review]
review changes

File: _posts/0000-01-02-AmkaE.md (hunk 1 above)
Comment (itzAmka) [inline _posts/0000-01-02-AmkaE.md]: "Your text"

**Review by itzAmka — COMMENTED** [review]
review changes

File: _posts/0000-01-02-AmkaE.md (hunk 1 above)
Comment (itzAmka) [reply _posts/0000-01-02-AmkaE.md]: replying

**Pull request merged**